qiskit/providers/models/jobstatus.py

```diff
@@ -41,8 +41,7 @@ def from_dict(cls, data):
                          :meth:`to_dict`.
 
         Returns:
-            qiskit.providers.model.JobStatus: The ``JobStatus`` from the input
-                dictionary.
+            JobStatus: The ``JobStatus`` from the input dictionary.
```

Comment:
Instead, should you move `JobStatus` to a type hint?

Reply:
I wasn't really prepared to have this PR apply type hints to a module that didn't previously contain them.  I think it'd be odd if we had a couple of the functions hinted and not all of them, and it would be too much for the scope of this PR if it both fixed the documentation and added new type hints everywhere.